decred_memes

```diff
@@ -0,0 +1,12 @@
+| Title               | decred meme collection |
+| Author              | OfficialCryptos |
+| Featured text       |Decred meme|
+
+Thesis statement for art collection - What the article is about, and what is it aiming to prove
+
+# Article Title
+
+Meme Piece 1
```

Comment:
![meme interstellar dcr](https://github.com/PhoenixGreen/Cypherpunktimes-articles/assets/158556488/bc9902e1-5595-44e8-8673-19aa7c0193bd)

Reply:
Is this image from a movie? Just a heads up, we can’t publish images that are subject to copyright E.g. anything from TV shows, movies, games etc.